src/main/java/com/capitalone/dashboard/auth/apitoken/ApiTokenRequestFilter.java

```diff
@@ -59,7 +60,7 @@ public Authentication attemptAuthentication(HttpServletRequest request, HttpServ
 
         String authHeader = request.getHeader("Authorization");
 
-        String encodedAuthStr = authHeader.substring(authHeader.indexOf(" "), authHeader.length());
+        String encodedAuthStr = CharMatcher.WHITESPACE.matchesAnyOf(authHeader) ? authHeader.substring(authHeader.indexOf(" "), authHeader.length()) : "";
```

Comment:
@nameisaravind  if there is no white space retrun authHeader instead of blank. This way flow will route to AuthenticationException.

Reply:
@nameisaravind  can you add test case for this change

Reply:
@nireeshT authHeader returned instead of "", tests added.